libfreerdp/core/rdp.c

```diff
@@ -1857,13 +1857,11 @@ void rdp_reset(rdpRdp* rdp)
 	license_free(rdp->license);
 	transport_free(rdp->transport);
 	fastpath_free(rdp->fastpath);
-	nla_free(rdp->nla);
```

Comment:
Same here, this is recreated to reset to initial state.

Reply:
no, this is an addition i added in  #5838. is was a mistake, i explained why. this struct is created in transport_connect_nla and does not need to be recreated here

Reply:
@kubistika oops, forgot ^^